_posts/0000-01-02-Amrutha05.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
This Was a Great beginning With Git Hub
Learned many things
explored new tools

Reply:
add few lines